packages/sdk/client/src/services/agent-hosting-provider.ts

```diff
@@ -44,7 +76,44 @@ export class DXOSAgentHostingProviderClient implements AgentHostingProviderClien
       baseURL: runtimeAgentHostingConfig.server,
       password: this._clientConfig.get('runtime.app.env.DX_AGENTHOSTING_PASSWORD'),
     };
-    log.info('AgentHostingProviderClient initialized', { config: this._config });
+  }
+
+  init(authToken?: any) {
+    if (!this._checkAuthorization(authToken)) {
+      return false;
+    }
+
+    if (!this._config.baseURL.endsWith('/')) {
```

Comment:
if needed move this check to the constructor, but it's odd to muatate the config that's passed in. Anyway URL constructor below should normalize the trailing slash.

Reply:
I'll move it, but the check is necessary as it won't append the path unless there's a trailing slash and we don't want to assume the caller will provide one. the [WhatWG](https://url.spec.whatwg.org/) API is pretty astonishing to me in this regard.